lib/validic/response.rb

```diff
@@ -1,9 +1,11 @@
 module Validic
   class Response
+    include REST::Request
     attr_reader :summary, :records, :attributes
-    def initialize(summary, response)
-      @summary = summary
 
+    def initialize(summary, response, client = nil)
+      @summary = summary
+      @client = client if client
```

Comment:
yes, but you added that. my question is why? 

Above you say that faraday needs a client connection but Im not seeing where in the code that is true. maybe better stated: what does having a connection method on the Response object provide us the ability to do>